test/IDE/complete_call_arg.swift

```diff
@@ -866,3 +860,31 @@ func testClosurePlaceholderPrintsTypesOnlyIfNoInternalParameterNamesExist() {
 // COMPLETE_CLOSURE_PARAM_WITHOUT_INTERNAL_NAMES-DAG: Decl[FreeFunction]/Local:           ['(']{#callback: (Int, Int) -> Bool##(Int, Int) -> Bool#}[')'][#Void#];
 // COMPLETE_CLOSURE_PARAM_WITHOUT_INTERNAL_NAMES: End completions
 }
+
+func testCompleteLabelAfterVararg() {
+  enum Foo {
+    case bar
+  }
+
+  struct Rdar76355192 {
+    func test(_: String, xArg: Foo..., yArg: Foo..., zArg: Foo...) {}
+  }
+
+  private func test(value: Rdar76355192) {
+    value.test("test", xArg: #^COMPLETE_AFTER_VARARG^#)
+    // COMPLETE_AFTER_VARARG: Begin completions
+    // COMPLETE_AFTER_VARARG-DAG: Pattern/ExprSpecific:               {#yArg: Foo...#}[#Foo#];
+    // COMPLETE_AFTER_VARARG-DAG: Pattern/ExprSpecific:               {#zArg: Foo...#}[#Foo#];
+    // COMPLETE_AFTER_VARARG: End completions
+    value.test("test", xArg: #^COMPLETE_AFTER_VARARG_WITH_PREV_PARAM^#)
+    // COMPLETE_AFTER_VARARG_WITH_PREV_PARAM: Begin completions
+    // COMPLETE_AFTER_VARARG_WITH_PREV_PARAM-DAG: Pattern/ExprSpecific:               {#yArg: Foo...#}[#Foo#];
+    // COMPLETE_AFTER_VARARG_WITH_PREV_PARAM-DAG: Pattern/ExprSpecific:               {#zArg: Foo...#}[#Foo#];
+    // COMPLETE_AFTER_VARARG_WITH_PREV_PARAM: End completions
```

Comment:
These tests don't look right. For `xArg: #^HERE^#`, expected suggestion are global expression completions.
Also, `xArg: #^COMPLETE_AFTER_VARARG_WITH_PREV_PARAM^#` is currently the same as `COMPLETE_AFTER_VARARG `. This probably meant `xArg: .bar, #^HERE^#` which expects argument labels _and_ global expressions.

Reply:
Of course 🤦  Updated the tests to test the right thing and the PR to behave according to the updated tests.